ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -753,6 +753,13 @@ literals:
   - DISCHARGING
   - STANDBY
 - battery_charge_percentage_sensor
+- max_battery_temperature_sensor
```

Comment:
`max` and `min` would require an `aggregation_descriptor` in this use case -- I am assuming you are trying to model the high and low temperature and voltage over a period of time?

Reply:
There are multiple internal temperature sensors and voltage sensors in the system. They are only providing the min and max values among all the sensors measured.

Reply:
thank you for clarifying :)